### DISCUSSION TITLE
Mirth Connect 2.2.3 documentation

### DISCUSSION THREAD

**Discussion by fjsalgado** [nextgenhealthcare/connect#5222] [Q&A]
Hello,
I have to develop some channels using mirth v.2.2.3 and I can't find any official documentation for this version.
Does anyone know how to find it?
Thanks in advance.

**Comment by jonbartels** [comment]
I could not readily locate the documentation. I checked:
* The MC download archives. I can find 2.2.3 binaries but not manuals https://mirthdownloadarchive.s3.amazonaws.com/index.html?prefix=connect/2.2.3.6825.b80/
* Changing the URL for a newer user guide to find an older version https://s3.amazonaws.com/downloads.mirthcorp.com/connect-user-guide/3.12.0/mirth-connect-user-guide.pdf (change 3.12.0 to 2.2.3)
* The forums https://forums.mirthproject.io/ I could not find the right search terms, you should try some searches there
* Here is the original release announcement for 2.2.3 - https://forums.mirthproject.io/forum/mirth-connect/general-discussion/9374-mirth-connect-2-2-3-released . There might be useful links in that thread, but I'm not sure if those links would still be live websites

Where did you look to find the documentation?

You really should consider installing a newer version of MC. 2.2.3 is almost a decade old and will operate differently than the 3.x series.

**Comment by fjsalgado** [comment]
Hello jon,
We work for a client that, for the moment, is not interested in migration, so we have to develop our new channels in this version. 
I understand that is a very old (and deprecated) version, but in this case our client leads.
I searched in general in google and in mirth web pages (even asked in the mirth forum).
We will implement the channels, even if we can't find the official documentation.
Anyway, thank you for your message.
Regards.
